api/controllers/game/rematch-gamestate.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Endpoint to request/decline playing again
+ * If both players accept rematch, creates
+ * a new game, switching who goes first
+ * 
+ * New game will be ranked/casual based on previous match
+ * with name "firstPlayerUsername VS secondPlayerUsername {p0wins}-{p1Wins}-{stalemates}"
+ */
+const gameAPI = sails.hooks['customgamehook'];
+module.exports = async function (req, res) {
+  try {
+    const { usr: userId } = req.session;
+    const { gameId: oldGameId } = req.params;
```

Comment:
pulling gameId from the req.params this way the call mimics the other "Moves".... `/api/game/:gameId/rematch`